.circleci/config.yml

```diff
@@ -645,6 +645,19 @@ jobs:
             docker push ${ROUTER_TAG}:${VERSION}-debug
             # Push release image
             docker push ${ROUTER_TAG}:${VERSION}
+      - run:
+          name: Helm build
+          command: |
+            # Install Helm
+            curl https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 | bash
+            # Package up the helm chart
+            helm package helm/chart/router
+            # Make sure we have the newest chart
+            CHART=$(ls -t router*.tgz| head -1)
+            # NB: GH token expires 30/3/2023
+            echo ${GITHUB_DOCKER_TOKEN} | helm registry login -u garypen --password-stdin ghcr.io
```

Comment:
Let's file a follow-up issue to ASAP replace this with a token created by the `apollo-bot2` user.  (And anywhere else we use it — l like the lines a little bit up from here).

Reply:
I filed: #1453